src/tools/rest-frontend/resources/structure.json

```diff
@@ -0,0 +1,1896 @@
+[
+    {
```

Comment:
JSON doesn't allow comments by standard, as far as I know (think we had this question before and I looked it up?).

Not so sure what you want me to explain, though.